app/assets/stylesheets/govuk-component/govspeak/_information-callout.scss

```diff
@@ -0,0 +1,22 @@
+// Govspeak information callout
+// http://govuk-component-guide.herokuapp.com/components/govspeak/fixtures/information_callout
+//
+// Support:
+//
+// - alphagov/whitehall: ✔︎
+// - alphagov/govspeak: ✔︎
+
+.govuk-govspeak {
+  .info-notice {
+    border-left: 1em solid $panel-colour;
+    padding: 1em 0 1em 1em;
+    margin: 2em 0;
+
+    // Remove margin from the last element
+    p:last-child,
```

Comment:
Good question. Probably not? It's hard to know exactly what elements are going to be used inside the callout and so what this could affect, but at the same time that's also a good argument for making it a generic `:last-child`…